code/modules/uplink/uplink_items/spy_unique.dm

```diff
@@ -121,3 +121,18 @@
 	name = "Syndicate First Medic Kit"
 	desc = "A syndicate tactical combat medkit, but only stocked enough to do basic first aid."
 	item = /obj/item/storage/medkit/tactical_lite
+
+/datum/uplink_item/implants/spy_unique/antistun
+	name = "CNS Rebooter Implant"
+	desc = "This implant will help you get back up on your feet faster after being stunned. Comes with an autosurgeon."
+	item = /obj/item/autosurgeon/syndicate/anti_stun/single_use
+
+/datum/uplink_item/implants/spy_unique/reviverplus
```

Comment:
But you use the name reviverplus here? Won't it get confusing?

Reply:
copypaste moment